mod-data-import-converter-storage-server/src/test/java/org/folio/rest/impl/association/JobToActionProfileTest.java

```diff
@@ -16,18 +24,24 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import java.util.UUID;
-
-import static org.hamcrest.Matchers.is;
 
 @RunWith(VertxUnitRunner.class)
 public class JobToActionProfileTest extends AbstractRestVerticleTest {
-  private static final String ASSOCIATION_TABLE_NAME = "job_to_action_profiles";
-  private static final String JOB_PROFILES_TABLE_NAME = "job_profiles";
-  private static final String ACTION_PROFILES_TABLE_NAME = "action_profiles";
+
+  private static final String ACTION_TO_MAPPING_PROFILES = "action_to_mapping_profiles";
+  private static final String JOB_TO_ACTION_PROFILES = "job_to_action_profiles";
+  private static final String JOB_TO_MATCH_PROFILES = "job_to_match_profiles";
+  private static final String JOB_PROFILES = "job_profiles";
+  private static final String ACTION_PROFILES = "action_profiles";
   private static final String JOB_PROFILES_URL = "/data-import-profiles/jobProfiles";
   private static final String ACTION_PROFILES_URL = "/data-import-profiles/actionProfiles";
   private static final String ASSOCIATED_PROFILES_URL = "/data-import-profiles/profileAssociations";
+  private static final String DETAILS_BY_MASTER_URL = "/data-import-profiles/profileAssociations/{masterId}/details";
+  private static final String MASTERS_BY_DETAIL_URL = "/data-import-profiles/profileAssociations/{detailId}/masters";
+  private static final String ACTION_TO_MATCH_PROFILES = "action_to_match_profiles";
```

Comment:
Is this 'ACTION_TO_MATCH_PROFILES' field really used your test cases ?

Reply:
Yes, it is used in clearTables method

Reply:
Why do you need to clean 'ACTION_TO_MATCH_PROFILES' table you if no test writes data there?

Reply:
The same for 'action_to_action_profiles' and 'match_to_action_profiles'

Reply:
I had issues with foreign keys when data were removed.
You are right. I double checked it and I gonna remove it as unnecessary calls.